tests/DatabaseTest.php

```diff
@@ -20,6 +20,28 @@
 
 	use TestCaseTrait;
 
+	/**
+	 * Creates .htconfig.php for bin/worker.php execution
+	 */
+	protected function setUp()
```

Comment:
`setUp()` is called before each test; I don't think that's necessary here.

You could do this instead:
```diff
@@ -23,9 +23,9 @@ abstract class DatabaseTest extends TestCase
        /**
         * Creates .htconfig.php for bin/worker.php execution
         */
-       protected function setUp()
+       public static function setUpBeforeClass()
        {
-               parent::setUp();
+               parent::setUpBeforeClass();
 
                $base_config_file_name = 'htconfig.php';
                $config_file_name = '.htconfig.php';
```
So it is called once before starting the testsuite.